pkg/kubectl/cmd/create.go

```diff
@@ -40,7 +40,7 @@ type CreateOptions struct {
 
 var (
 	create_long = templates.LongDesc(i18n.T(`
```

Comment:
nit: how about also change coding style here ? also I'm not sure if there's any other places exist coding style like this in kubectl